.github/workflows/scheduled.yaml

```diff
@@ -0,0 +1,49 @@
+# Tests in this workflow are run periodically.
+# They are generally very expensive and not on the critical path
+# for development
+# scheduled workflows only run in the default branch
+
+name: Scheduled Tests
+
+on:
+  schedule:
+    - cron: "42 3 * * 0"
+
```

Comment:
In practice this hasn't happened in the past 5 years, but I can see how there could be some incentive. Please keep in mind that there is a (not striclty enforced, but to be respected) limit of two scans a week

Reply:
> In practice this hasn't happened in the past 5 years

In reality, running Coverity Scan on demand happened many times in the past year or so. AFAIK, it was not done using Jenkins (for arguably obvious reasons), but that should not matter -- the _need_ for such tests does exist.

Running Coverity workflow on demand was also done once in the past 5 hours as I was testing various aspects of the proposed workflow. I am pretty sure others working on this workflow in the future may enjoy the ability to trigger the workflow on demand.

> Please keep in mind that there is a (not striclty enforced, but to be respected) limit of two scans a week

I hope we will continue to respect reasonable limits even when given the tool to trigger the workflow on demand. If we can add a reference to documentation suggesting that limit, it may be a good idea.